<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWidgetMapper</class>
 <widget class="QWidget" name="DataWidgetMapper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataWidgetMapper</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="deviceTableView"/>
   </item>
   <item row="1" column="0">
    <widget class="QDockWidget" name="dockWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>105</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>524287</width>
       <height>105</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="floating">
      <bool>false</bool>
     </property>
     <property name="windowTitle">
      <string/>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableView" name="deviceTableView_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
